script.js

```diff
@@ -0,0 +1,79 @@
+/* add todo*/
+const todoInput=document.getElementById("todo-input");
+const addTodoButton=document.getElementById("todo-button");
+const doingTodoList=document.getElementById("doing");
+const doneTodoList=document.getElementById("done");
+let todoArray=[];
+const doingCount=document.getElementById("doing-count");
+const doneCount=document.getElementById("done-count");
+
+/*create todo li*/
+function createTodoList(todoObj){
+    const todoListItem=document.createElement("li");
+    const deleteImg=document.createElement("img");
+    todoListItem.innerText="🐈 "+todoObj.text;
+    todoListItem.id=todoObj.id;
+    todoListItem.addEventListener("click", toggleTodo);
+    deleteImg.src="./img/bin.png";
+    deleteImg.addEventListener("click", deleteTodo);
+    todoListItem.appendChild(deleteImg);
+    todoObj.type? doingTodoList.appendChild(todoListItem):doneTodoList.appendChild(todoListItem);
+}
+
+/*click add todo button*/
+function onClickAddTodo(e){
```

Comment:
대상 - 동작 순으로 네이밍을 하는 것이 문맥상 맞고 일반적인 컨벤션이라고 합니다!
```suggestion
function onAddTodoButtonClick(e){
```

Reply:
쓰면서도 뭔가 어색한데..?라는 생각이 있었는데 이래서였군요ㅋㅋㅋㅋ ㅎㅎ..

Reply:
저는 개인적으로 조금 더 길더라도 다음과 같이 쓰는게 좋을 것 같아요~
```suggestion
function handleAddTodoClicked(e){
```